.github/actions/test-win/action.yml

```diff
@@ -25,10 +25,18 @@ runs:
   steps:
 
     - name: Download built test directory
-      uses: actions/download-artifact@v3
+      uses: actions/download-artifact@v4
       with:
         name: ${{ inputs.build-artifact }}
 
+    - name: Setup miniconda
+      uses: conda-incubator/setup-miniconda@v3
+      with:
+        auto-update-conda: true
+        activate-environment: tigl-bld
+        environment-file: environment.yml
+        auto-activate-base: false
+
```

Comment:
Why are those extra miniconda options set for windows and not for the other OS?

Reply:
Good find! I added them to debug a failing windows test job while this is in progress. It is not needed and I removed it.